### PR TITLE
Resize comparison table with slider

### PR DESCRIPTION
**Pull request by sroy3**
Part of #2585 

https://user-images.githubusercontent.com/3683420/225697074-cd1feebc-e5df-4245-a9ec-9888245196a0.mov

### CODE REVIEW

**Comment by sroy3** [comment]
Sorry for the renaming, it just made more sense that way. The main part of the change is found in `ComparisonTableWrapper.tsx`, `ComparisonTable.tsx`, `PlotsContainer.tsx` and `plots/components/styles.modules.scss`.

**Comment by sroy3** [comment]
Won't be necessary after all. I'll revert the change in the submodule.

**Comment by mattseddon** [comment]
Sticky slider collides with comparison header:

image

Can the slider/ribbon fill the space on the RHS?

image

Will both of the above be fixed by the section having its own scrollbar?

File: extension/src/plots/model/index.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ export type CustomPlotsOrderValue = { metric: string; param: string }
 export class PlotsModel extends ModelWithPersistence {
   private readonly experiments: Experiments
 
-  private nbItemsPerRow: Record<PlotsSection, number>
+  private nbItemsPerRowOrWidth: Record<PlotsSection, number>
```
Comment (mattseddon) [inline extension/src/plots/model/index.ts:59]: [Q] Do these need to be stored together?

**Review by mattseddon — APPROVED** [review]
Change looks fine outside of previously mentioned issue

File: extension/src/plots/model/index.ts (hunk 1 above)
Comment (sroy3) [reply extension/src/plots/model/index.ts:59]: Was debating this, but they function the exact same way it's just that one cannot be called `nbItemsPerRow`.  Separating them would probably mean checking whether it has one property or another multiple times.

**Review by julieg18 — APPROVED** [review]

**Comment by sroy3** [comment]
> Sticky slider collides with comparison header:
> 
> image
> 
> Can the slider/ribbon fill the space on the RHS?
> 
> image
> 
> Will both of the above be fixed by the section having its own scrollbar?
> 
> Edit: With pinned column
> 
> image

The sticky header colliding is easy to fix now. Same for the pinned column. The header taking the full width will be easier with the local scrollbar (next item on my list), else we'll end up with sliders past smaller sections.

**Pull request merged**